internal/client/client.go

```diff
@@ -6,41 +6,47 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	. "github.com/mezmo/terraform-provider-mezmo/internal/provider/models/modelutils"
+	"golang.org/x/net/context"
 )
 
 type Client interface {
-	Pipeline(id string) (*Pipeline, error)
-	CreatePipeline(pipeline *Pipeline) (*Pipeline, error)
-	UpdatePipeline(pipeline *Pipeline) (*Pipeline, error)
-	DeletePipeline(id string) error
+	Pipeline(id string, ctx context.Context) (*Pipeline, error)
+	CreatePipeline(pipeline *Pipeline, ctx context.Context) (*Pipeline, error)
+	UpdatePipeline(pipeline *Pipeline, ctx context.Context) (*Pipeline, error)
+	DeletePipeline(id string, ctx context.Context) error
+
```

Comment:
Sorry about all these changes.  We needed to start passing `ctx` through to the client so that we can do proper logging with `tflog`.  That's the only way to see logs when running the provider.